tasks/main.yml

```diff
@@ -1 +1,31 @@
 # role tasks
+
+- name: Install necessary packages
+  package:
+    name: parted
+    state: present
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+
+- name: Clear the disk signatures
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "wipefs --all --force {{ item }}"
+  register: signatures_before
+  changed_when: '"offset" in signatures_before.stdout'
```

Comment:
Given that this is running unconditionally, and we're using the `--force` flag to `wipefs`, and that this role is not idempotent, we might as well just report 'changed' status here.

Reply:
I mean, if we can detect that something has or has not changed, we should. But are we sure that this conditional is correctly picking up whether something has changed?

Reply:
updated as my intent is to run unconditionally